packages/utilities/src/dateTimeFormat/absolute.ts

```diff
@@ -68,6 +73,7 @@ export function formatRange(
     style: Intl.DateTimeFormatOptions['timeStyle'];
     timeStyle: Intl.DateTimeFormatOptions['timeStyle'] | null;
     dateStyle: Intl.DateTimeFormatOptions['dateStyle'] | null;
+    timeZone?: Intl.DateTimeFormatOptions['timeZone'];
   }>
```

Comment:
`options` is already wrapped in `Partial<>`, so the `?` after `timeZone` is redundant.